python/link_prediction/link_prediction_util.py

```diff
@@ -0,0 +1,179 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import itertools
+import numpy as np
+import scipy.sparse as sp
+import dgl
+from dgl.data import CoraGraphDataset
+from sklearn.metrics import roc_auc_score
+from typing import Tuple
+from models.GraphSAGE import GraphSAGE
```

Comment:
if you want to write:

`from models import GraphSAGE`

set it in the folder's __init__.py, similar to here: https://github.com/memgraph/mage/blob/main/python/mage/graph_coloring_module/__init__.py